packages/breadcrumbs/src/styled/StyledChevronIcon.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import ChevronRightStrokeIcon from '@zendeskgarden/svg-icons/src/12/chevron-right-stroke.svg';
+
+const LTR_COMPONENT_ID = 'breadcrumbs.chevron_right_stroke_icon';
+const RTL_COMPONENT_ID = 'breadcrumbs.chevron_left_stroke_icon';
+
+const getGardenId = (rtl: boolean) => (rtl ? RTL_COMPONENT_ID : LTR_COMPONENT_ID);
+
+/**
+ * Accepts all `<svg>` props
+ */
+export const StyledChevronIcon = styled(ChevronRightStrokeIcon).attrs(props => ({
+  'data-garden-id': `${getGardenId(props.theme.rtl)}`,
```

Comment:
The `getGardenId` utility returns a string, you shouldn't wrap this in a template.

Reply:
I removed this code entirely because of JZ's comments pointed out that this was unnecessary (which sounds fair to me).  I will go ahead and mark this as resolved.